mainsail/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: mainsail_web_1
+      APP_PORT: 80
+  web:
+    image: ghcr.io/mainsail-crew/mainsail:sha-4c86d7c@sha256:960b63d7f1c438ab68b0add4a5eac0aa972ec29b13e3ec16ccbc2dfadf86d568
+    restart: on-failure
```

Comment:
Can we try adding `user: "1000:1000"` here so that the container runs as non-root and see if everything still works?

It's not a deal-breaker or anything, it's just nice to run as non-root when not strictly required.

Reply:
Ah sorry forgot to post that. Already tried, does not work sadly. Fails to start with a lot of permission errors.

Reply:
ah darn. Thanks for looking into that!